lib/kcerts/ssh.go

```diff
@@ -96,9 +97,27 @@ func (a SSHAgent) Valid() bool {
 	return err == nil
 }
 
+func (a SSHAgent) AddCertificates(privateKey *rsa.PrivateKey, publicKey ssh.PublicKey, ttl uint32) error {
+	conn, err := net.Dial("unix", a.Socket)
+	if err != nil {
+		return err
+	}
+	defer conn.Close()
+	cert, ok := publicKey.(*ssh.Certificate)
+	if !ok {
+		return fmt.Errorf("public key is not a valid ssh certificate")
+	}
+	agentClient := agent.NewClient(conn)
```

Comment:
I would rather not do this right now because it would affect other parts of Close() functionality, including  when it gets reassigned. I like it how it is right now where it cannot leak open connections/all commands op -> run -> finish.

Reply:
sgtm